themes/10up-theme/assets/css/frontend/tailwind/reset.css

```diff
@@ -0,0 +1,241 @@
+/*
+Custom Reset modified for WordPress based off of Tailwind's reset:
```

Comment:
@Nikki-Jones Can you elaborate a bit on what changes you made to be more suitable for WordPress? I haven't really encountered this before but I'm interested to understand.

Reply:
@iansvo the explanation is here - https://github.com/10up/wp-scaffold/blob/3eb3750d7c3bd8944c500f8022f4b712660ab66a/themes/10up-theme/tailwind.config.js, but let me know if you think it's worth adding in both spots